### ISSUE TITLE
(1P) FSE: Change to 2 column layout for Templates on small screens

### ISSUE THREAD

**Issue by getdave** [Automattic/wp-calypso#34174]
Currently, on smaller screens, the Template layout is 1 column. When loading the Modal it looks broken because you only see the "Blank Option".

See `p58i-7RD-p2#comment-42225`

cc @iamtakashi @fditrapani

**Comment by obenland** [comment]
We could also consider switching to landscape images through srcset, when we snap to single column

**Comment by marekhrabe** [comment]
I looked at this before realizing it's assigned. Leaving my experimental code that I tried as a possible inspiration:

```
.template-selector-control__options {
	grid-gap: 0;
	
	@media screen and ( max-width: 460px ) {
		grid-template-columns: repeat( 2, minmax( 1px, 1fr ) );
	}
}
```

I have changed the `grid-gap` globally (not only for mobile) as I think it doesn't serve much purpose when we already have a lot of padding inside individual tiles. This will allow us to show a bit bigger previews on the equal screen real estate.

Leaving this now :disappear:
